src/main/java/fi/otavanopisto/kuntaapi/server/integrations/mikkelinyt/MikkeliNytEventProvider.java

```diff
@@ -59,15 +56,15 @@
   
   @Inject
   private MikkeliNytImageLoader mikkeliNytImageLoader;
-
-  @Inject
-  private ImageReader imageReader;
-
-  @Inject
-  private ImageWriter imageWriter;
-  
-  @Inject
-  private ImageScaler imageScaler;
+//
+//  @Inject
+//  private ImageReader imageReader;
+//
+//  @Inject
+//  private ImageWriter imageWriter;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)